### ISSUE TITLE
Pattern for phenotype extension

### ISSUE THREAD

**Issue by matentzn** [obophenotype/upheno#591]
Use case: we need a way to represent

1) cases where only a part of the phenotype definition in known
2) where we want to define a phenotype in terms of another, undefined phenotype which is then extended.

**Comment by matentzn** [comment]
Please everyone give examples if you have such.

**Comment by dosumis** [comment]
@matentzn - I think the ticket needs some more details & motivating examples. Phenotype extension sounds like post-composition (phenotypes  extended during annotation), but I think you actually mean phenotypes defined in terms of other phenotypes?  

When you talk about 'undefined phenotype' or 'where only part of the phenotype definition is known'  I assume you refer to 'logical definitions' AKA equivalent Class assertions?  The details are presumably perfectly known to biologists, we just don't know how to fully formalize (?)

**Comment by matentzn** [comment]
Yes exactly: I am only talking about logical; The case where I dont have the expressive means to describe the necessary and sufficient conditions, but can say some necessary ones. So to be clear: **_phenotype extension is the process of refining an existing logical definition of a phenotype to make it more specific_**.

This is an example: [Hepatic encephalopathy](https://www.ebi.ac.uk/ols/ontologies/hp/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FHP_0002480&viewMode=All&siblings=false)
```
Central nervous system dysfunction in association with liver failure and characterized clinically (depending on degree of severity) by lethargy, confusion, nystagmus, decorticate posturing, spasticity, and bilateral Babinski reflexes. [ HPO:probinson ]
```

We know that this phenotype inheres in the liver, and that it can be fully describe as an Encephalopathy that inheres in the liver. However, we lack the expressive power to describe Encephalopathy. 

Note that in this case, the definitions are a bit weird for an noob like me:

```
Encephalopathy is a term that means brain disease, damage, or malfunction. In general, encephalopathy is manifested by an altered mental state. [ HPO:probinson KI:phemming ]
```

Lets not discuss this phenotype biologically now (defs appear a bit weird to a noob like me), but acknowledge that there are cases like this where OWL has a hard time to describe the parent phenotype, but a child can be described as a refined extension of the parent - hence the need for what I call phenotype extension.
